### PR TITLE
added dist to .gitignore and fixed license date

### PR DESCRIPTION
**Pull request by bugyboo**
Just organized the package and Added build files to in-line Javascript and transpile typescipt code to be able to use this package for modern Javascript frameworks like angular and react..

### CODE REVIEW

**Comment by bugyboo** [comment]
I'm thinking in future development is to rewrite the script on TypeScript for many reasons.
the benefits of transforming to this approach are code will be more efficient and less error prune, also, we will be able to use tests framework.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019
```
Comment (xsoh) [inline LICENSE]: The year should be 2017

File: dist/solarHijri-js.common.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
+/*!
```
Comment (xsoh) [inline dist/solarHijri-js.common.js:1]: the directory `dist/` shall be ignored in the git repo

Comment (bugyboo) [reply dist/solarHijri-js.common.js:1]: +1 
I agree

**Pull request closed without merging**